### DISCUSSION TITLE
对种子文件的统一管理和视频重命名

### DISCUSSION THREAD

**Discussion by 88866888** [jxxghp/MoviePilot#2920] [建议]
在下载过程中，MP可以很好的下载想要的电影和电视节目，但来源不同就造成下载的种子文件名和视频文件名各有不同，当然在同一发布者的视频会很好的下载到同一文件夹下，这样对不是很会操作Mp的来说是很难管理的一件事，因为在下载后会有很多个种子文件夹。如果MP能在下载前就把种子文件命名为相应的统一文件名，并将相应的种子视频文件在下载完成后改名成相应的文件夹名。这样就能很好的管理了。我知道Mp有词表来解决

**Comment by dlovew** [comment]
其实你可以看看有个插件叫订阅规则自动填充
